File: support_tools/swigwin-3.0.12/Examples/test-suite/php/import_nomodule_runme.php
<?php
require "tests.php";
require "import_nomodule.php";

// No new functions
check::functions(array(create_foo,delete_foo,test1,is_python_builtin));
// No new classes
check::classes(array(import_nomodule,Bar));
// now new vars
check::globals(array());

$f = import_nomodule::create_Foo();
import_nomodule::test1($f,42);
import_nomodule::delete_Foo($f);

$b = new Bar();
import_nomodule::test1($b,37);

check::done();
?>

File: support_tools/swigwin-3.0.12/Examples/test-suite/python/import_nomodule_runme.py
from import_nomodule import *

# This test is expected to fail with -builtin option.
# The base class is needed for the builtin class hierarchy
if is_python_builtin():
    exit(0)

f = create_Foo()
test1(f, 42)
delete_Foo(f)

b = Bar()
test1(b, 37)
